README.md

```diff
@@ -1,27 +1,62 @@
+        <img alt="Build" src="https://img.shields.io/github/actions/workflow/status/fuzzylabs/matcha/ci.yml">
+    </a>
+    <a href="https://github.com/fuzzylabs/matcha/blob/main/LICENSE">
+        <img alt="License" src="https://img.shields.io/github/license/fuzzylabs/matcha?color=blue">
+    </a>
+    <a href="https://github.com/fuzzylabs/matcha/releases">
+        <img alt="Release" src="https://img.shields.io/github/v/release/fuzzylabs/matcha?color=green">
+    </a>
+    <a href="https://pypi.org/project/matcha-ml/">
+        <img alt="Downloads" src="https://img.shields.io/pypi/dm/matcha-ml?color=orange">
+    </a>
+</p>
 
-Data Scientists and practitioners build ML pipelines and productionize them with the following capabilities:
+<h3 align="center">
+    <p>A tool for provisioning an MLOps environment on Azure in 20 minutes</p>
+</h3>
 
-* A way to run model training pipelines
-* A way to track experiments
-* A way to deploy and serve models
+&#127861; `matcha` is the open source tool for provisioning MLOps environments to Azure
 
-The current version of `matcha` provisions the above with a single command, using sensible defaults for the infrastructure.
+With a single command and using sensible defaults, Data Scientists and ML practitioners are provided with the following capabilities:
 
-More components could be added here, for example, managing and versioning datasets, managing models (registration and governance), and monitoring models. Adding all of these is on our roadmap.
+* &#127939; A way to run model training.
+* &#128099; A way to track experiments.
+* &#127869;&#65039; A way to deploy and serve models.
 
-`matcha` uses open source to solve this problem.
+&#127861; `matcha` uses open source tools to build this MLOps environment on Azure.
 
-By default, `matcha` will `provision` an MLOps environment to Azure with sensible defaults - enough for you to get going with cloud-based training and deployment. From there, you can use `matcha` to `get` information about your provisioned environment.
+<p align="center">
+GIF
+</p>
 
-Please see the following for more information, including install instructions, documentation, and how you can contribute to `matcha`:
+## 	&#128678; Getting Started
 
-* [Getting Started](https://fuzzylabs.github.io/matcha/getting-started/)
-* [Documentation](https://fuzzylabs.github.io/matcha/)
-* [Contributing](CONTRIBUTING.md)
+Keen to get started with `matcha`? We have a great getting started guide which you can find [here](https://mymatcha.ai/getting-started/).
 
-## Licence
+By the end, you'll understand how to use `matcha` and have a fully provisioned movie recommendation system in Azure, ready to serve some movie ratings!
+
+## &#128214; Documentation
+
+Want to learn more about `matcha`? See our [documentation](https://mymatcha.ai/) which covers what `matcha` is in more detail, how to use it, what Azure permissions are required, and much more!
+
+## &#128506;&#65039; Roadmap
```

Comment:
This is missing a link as we've not created a public-facing roadmap yet. I can be added in once we've done that.